### ISSUE TITLE
When I run the Base_OC_Module at a certain iteration, it will stop.

### ISSUE THREAD

**Issue by ycszen** [PkuRainBow/OCNet#16]
`Pytorch version: 0.4.1`
`GPU benchmark: 8 x 1080 Ti`
`CUDA version: 9.0`

Experiment config:
`batch size: 16`
`num workers: 16`
`input size: 480x480`

When I use the Base_OC_Module on the ADE20K dataset, my experiment will stop at a certain iteration without other notion output. Did you have the similar experience?

**Comment by PkuRainBow** [comment]
Sorry, I think it can be a bug of inplace-abn. Please refer to https://github.com/mapillary/inplace_abn/issues/58.

We use 4 x P100 for all the experiments.

**Comment by ycszen** [comment]
Yes. When I use 4 x 1080Ti, all of it becomes normal. This should be a bug of inplace-abn. Thank you.

**Comment by KeyKy** [comment]
I get the same problem and it only work for 4 gpus.
